src/components/Media/Media.jsx

```diff
@@ -31,7 +30,6 @@ const Gallery = () => {
 
   const [imageDisplay, setImageDisplay] = useState(false);
   const [imageToShow, setImageToShow] = useState(``);
-  const [blur, setblur] = useState(``);
```

Comment:
You removed the hook but forgot to remove associated event handler `setblur`.

Reply:
done